scripts/generate_dump

```diff
@@ -876,6 +899,17 @@ collect_mellanox() {
         done
         rm -rf /tmp/sdk-dumps
     fi
+
+    # run 'hw-management-generate-dump.sh' script and save the result file
+    HW_DUMP_FILE=/usr/bin/hw-management-generate-dump.sh
+    if [ -f "$HW_DUMP_FILE" ]; then
+      ${CMD_PREFIX}${timeout_cmd} /usr/bin/hw-management-generate-dump.sh $ALLOW_PROCESS_STOP
```

Comment:
if timeout expired and file does not exist, we should cont but IMO should have an error message identify timeout expired and no hw mgmt file. .

Reply:
Fixed